cmd/query/main_test.go

```diff
@@ -366,19 +365,16 @@ func TestMetadataHappyHTTPResponse(t *testing.T) {
 		json.Unmarshal([]byte(expectedMetadata), &expectedMap)
 		json.Unmarshal([]byte(metadata), &actualMap)
 
-		val, ok := actualMap["importTimeStamp"]
-		if !ok {
-			t.Errorf("Metadata not equal in case '%s'", testcase.name)
+		if _, ok := actualMap["importTimeStamp"]; !ok {
+			t.Errorf("importTimeStampt is not found in case '%s'", testcase.name)
 		}
-		actualVal   := val.(string)
-		expectedVal := expectedMap["importTimeStamp"].(string)
-		if !regexp.MustCompile(expectedVal).MatchString(actualVal) {
-			t.Errorf("Metadata not equal in case '%s'", testcase.name)
-		} else {
-			expectedMap["importTimeStamp"] = "dummy"
-			actualMap["importTimeStamp"]   = "dummy"
-		}
-		require.Equal(t, expectedMap, actualMap, "Metadata not equal in case '%s'", testcase.name)
+
+		require.Regexp(t, expectedMap["importTimeStamp"], actualMap["importTimeStamp"])
+
+		expectedMap["importTimeStamp"] = "dummy"
+		actualMap["importTimeStamp"]   = "dummy"
+
+		require.Equal(t, expectedMap, actualMap, "importTimeStampt is not found in case '%s'", testcase.name)
```

Comment:
And here error message "Metatadata not equal..." was more correct :)

Reply:
done :) https://github.com/equinor/vds-slice/pull/172/files#diff-079081f30f9137f135638d9b5e2a772d22a7232c750ab2dc087141318c8e168fR377